internal/transform/transform.go

```diff
@@ -126,7 +126,7 @@ func ExtractStyles(doc *astro.Node) {
 				return
 			}
 			// prepend node to maintain authored order
```

Comment:
Maybe we should update the comment too
```suggestion
			// append node to maintain authored order
```